conf/sample.ini

```diff
@@ -775,6 +775,12 @@
 # If true, assume role will be enabled for all AWS authentication providers that are specified in aws_auth_providers
 ; assume_role_enabled = true
 
+# Specify max no of pages to be returned by the ListMetricPages API
```

Comment:
is this list metrics page limit intentional?

Reply:
I noticed it was missing in the sample.ini, I have been told the sample.ini should match the default.ini, so I added it as well